### ISSUE TITLE
The package triqler.distribution is broken

### ISSUE THREAD

**Issue by fabianegli** [statisticalbiotechnology/triqler#19]
The package ["triqler.distribution"](https://github.com/statisticalbiotechnology/triqler/blob/master/setup.py#L32) defined in setup.py is broken.

```
python -m triqler.distribution
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/eglif/github/fabianegli/triqler/triqler/distribution/__main__.py", line 6, in <module>
    from .distribution import main
ModuleNotFoundError: No module named 'triqler.distribution.distribution'
```

**Comment by MatthewThe** [comment]
Thank you for reporting this issue.

`triqler.distribution` itself cannot be run as a standalone module.

You should be able to run its submodules though, e.g. 
 
```
python -m triqler.distribution.plot_hyperparameter_fits <triqler_input_file>
```

https://github.com/statisticalbiotechnology/triqler/wiki/Hyperparameters

**Comment by fabianegli** [comment]
If that is so, there should be no file `triqler/triqler/distribution/__main__.py` at all and the packages in setup.py should point to the actual runnable packages:

https://github.com/statisticalbiotechnology/triqler/blob/e91f3589caba3bce095412fe2df2f2c6863a4d7f/setup.py#L32

So `"triqler.distribution"` should be removed and `"triqler.distribution.plot_hyperparameter_fits"` added to make the package reflect the intended use.

**Comment by MatthewThe** [comment]
Yes, you're right. I will fix it.

**Comment by MatthewThe** [comment]
Fixed by 4cd5ae5 and 8744e2a
